builtins/src/string.rs

```diff
@@ -315,6 +315,7 @@ fn char_whitespace(vm: &mut SloshVm, registers: &[Value]) -> VMResult<Value> {
     }
 }
 
+#[allow(rustdoc::broken_intra_doc_links)]
```

Comment:
allows the doc to keep the '[' and ']' symbols in the slosh doc w/o rust doc complaining